timeline.md

```diff
@@ -9,26 +9,58 @@ css_id: timeline
 browser, Justin Samuel and Justin Cappos collaborate to design and publish an
 academic research paper on The Update Framework (TUF).
 
+**2011**: TUF Project moves to New York University Polytechnic
+(later NYU Tandon School of Engineering) when Justin Cappos accepts a post
+as an assistant professor in the Brooklyn, NY, school.
+
 **2013**: Justin Cappos, Trishank Kuppusamy, and Vladimir Diaz begin research
 into adapting and improving TUF for Python, Ruby, and other environments used
 for cloud computing.
 
+**2013**: The first of two Python Enhancement
+Proposals (PEP 458) dealing with TUF is published. "Surviving a Compromise of
+PyPI" details the integration of TUF into PyPI, the Python
+package manager.
+
 **2014**: Flynn.io independently implements TUF in the Go programming language.
 
-**2015**: Docker launches Notary. It also launches Docker Content Trust, which
-uses Notary to sign and verify container images.
+**2014**: PEP 480, a maximum security version of PEP 458, is published
```

Comment:
The entry doesn't end with a period, like the others.

Reply:
fixed